### DISCUSSION TITLE
Avoid ESP32 Arduino library versions 2.0.10 and 2.0.11

### DISCUSSION THREAD

**Discussion by corrados** [corrados/edrumulus#101] [General]
In ESP32 Arduino library versions 2.0.10 and 2.0.11, there is an issue with the available heap space which leads to a problem in Edrumulus. If you use these library versions and compile/upload Edrumulus to the ESP32, the edrumulus_gui.py will not start correctly since the MIDI transfer does not work correctly.

Since the newest versions 2.0.12 and 2.0.13 work fine with Edrumulus, I will now remove the warning message in the README file which was the following message:

⚠️ **WARNING: You have to use Ardunio board manager "ESP32 by Espressif Systems" VERSION 2.0.9 (i.e., do not use the latest releases 2.0.10 and 2.0.11) to get edrumulus working as expected (newer versions have issues with the MIDI transfer, see https://github.com/espressif/arduino-esp32/issues/8471).**

**Comment by 3hhh** [comment]
It might make sense to update it in the platformio config then as well.
